lib/queue_classic_matchers.rb

```diff
@@ -6,8 +6,8 @@
 module QueueClassicMatchers
   # Your code goes here...
   module QueueClassicMatchers::QueueClassicRspec
-    def self.find_by_args(queue_name, method, args, table = "queue_classic_jobs")
-      q = "SELECT * FROM #{table} WHERE q_name = $1 AND method = $2 AND args::text = $3"
+    def self.find_by_args(queue_name, method, args)
```

Comment:
This isn't backward compatible, so make sure you make this a minor release, not just a patch release.